src/client/stateManagers/box/states.js

```diff
@@ -78,6 +79,15 @@ export const fetchedPublicProfileErr = (state, error) => ({
   error,
 })
 
+export const noPublicProfileFound = (state, error) => ({
+  ...state,
+  loadingPublicProf: false,
+  loadedPublicProf: false,
+  loadedPublicProfSuccess: false,
```

Comment:
Think we should set these state variables as though the profile was loaded 

loading: false
loaded: true
loadedSuccess: true

Reply:
Makes sense.

Reply:
`loadedPublicProfileSuccess` i think should be marked true - because this state appears to be an error

Reply:
@rkzel this is still not fixed

Reply:
`useLinkedData` will try to format public profile if `loadedPublicProfileSuccess` is set to true.